spec/operations/story_operations_spec.rb

```diff
@@ -359,10 +360,70 @@
           ],
           mattermost: "[Test Project] The story ['Test Story']" \
             "(http://foo.com/projects/123#story-#{story.id}) has been rejected."
-        )
-
+          )
         subject.call
       end
     end
+
+    describe '::ReadAll' do
+      let(:user)           { create(:user, :with_team) }
+      let(:current_team)   { user.teams.first }
+      let(:project) { create(:project, :with_past_date, users: [user], teams: [current_team]) }
+      let(:pundit_context) { PunditContext.new(current_team, user, current_project: project) }
+      let(:policy_scope)   { Pundit.policy_scope(pundit_context, done_story) }
+
+      let(:done_story_params) do
+        {
+          title: 'Done story',
+          state: 'accepted',
+          estimate: 8,
+          created_at: DateTime.current.days_ago(10),
+          accepted_at: DateTime.current.days_ago(10),
+          started_at: DateTime.current.days_ago(10)
+        }
+      end
+
+      let(:active_story_params) do
+        {
+          title: 'Active story',
+          state: 'started',
+          started_at: DateTime.current.days_ago(2)
+        }
+      end
+
+      subject { StoryOperations::ReadAll.call(story_scope: policy_scope, project: project) }
+
+      context 'when there are stories in the done column' do
+        let(:done_story)      { project.stories.create!(done_story_params) }
```

Comment:
Can these be the `ReadAll`'s `describe`? It's probably gonna be used in other tests if needed and you're using it in the `policy_scope`'s  `let`, it's better to move it to there to avoid indirection while reading the tests.

Also, It seems to me that we should be using `let!` instead of `let` here. The value of a `let` will be only called when you reference the variable with its name and I don't see it being called _before_ the operation is ran in the last test, so this story is not even being created before calling the operation.